cassandra/cqlengine/columns.py

```diff
@@ -72,6 +73,7 @@ def get_property(self):
             return property(_get, _set)
 
 
+@total_ordering
```

Comment:
We still support Python 2.6. This was added in 2.7. We may need to think of alternatives.